src/main/java/org/ggp/base/util/game/LocalGameRepository.java

```diff
@@ -226,7 +226,10 @@ private static int getMaxVersionForDirectory(File theDir) {
                 } else if (theFile.getName().endsWith(".png")) {
                     // TODO: Handle other binary formats?
                     return readBinaryFile(theFile);
-                } else if (theFile.getName().endsWith(".xsl")) {
+                } else if (theFile.getName().endsWith(".jpg")) {
+                    return readBinaryFile(theFile);

```

Comment:
You'll need to make a similar change in BaseRepository.java in the ggp-repository project, so that this continues to work when the game is uploaded to the GGP.org online repository.